src/app/shared/assets.service.ts

```diff
@@ -818,7 +818,7 @@ export class AssetService {
                         let results = res
                         data.thumbnails = results['items'] // V1 ?items from search
                         // For multi-view items, make the thumbnail urls and update the array
-                        data.thumbnails = data.items.map((item) => {
+                        data.thumbnails = data.items.slice(pageStart, pageEnd).map((item) => {
```

Comment:
Worth noting that "slice" is a nice function since its failure state if items = [] would be to just return []